followthemoney/export/csv.py

```diff
@@ -0,0 +1,21 @@
+import csv
+
+
+def write_entity(fh, entity, extra_fields=dict()):
+    fieldnames = [prop.label for prop in entity.schema.sorted_properties]
+    fieldnames = ['id'] + list(extra_fields.keys()) + fieldnames
```

Comment:
I assume extra fields is stuff like the UI url in Aleph? Very cool. But is there a risk with the method-default dict on L4? Maybe let's make this None and then zero-guard it.

Reply:
Yeah, the default needs to be `None`. I mistakenly thought making it `dict()` instead of `{}` will help.